ensime_shared/typecheck.py

```diff
@@ -13,6 +13,14 @@ def buffer_typechecks(self, call_id, payload):
         if self.currently_buffering_typechecks:
             for note in payload['notes']:
                 self.buffered_notes.append(note)
+            self.editor.display_notes(self.buffered_notes)
+
+    def buffer_typechecks_and_display(self, call_id, payload):
+        """Adds typecheck events to the buffer, and displays them right away.
+        This is currently used as a workaround for issue https://github.com/ensime/ensime-server/issues/1616
+        """
+        self.buffer_typechecks(call_id, payload)
+        self.editor.display_notes(self.buffered_notes)
```

Comment:
Why is this line required? `buffer_typechecks` already displays notes. I think L16 or L23 could go away!